src/main.py

```diff
@@ -0,0 +1,64 @@
+import graph
+import pygame
+import draw
+import env
+
+DISPLAY_WIDTH = 800
+DISPLAY_HEIGHT = 600
+PATH = env.get_graph_data_path()
```

Comment:
This is not a constant (because its value will be evaluated at the start), so I would prefer to make it more like a variable inside `try` block below